src/game_state/player.rs

```diff
@@ -82,7 +82,7 @@ pub fn start_player_state(
                 broadcast_packet!(
                     messenger,
                     Packet::EntityLookAndMove(EntityLookAndMove {
-                        entity_id: player_clone.conn_id,
+                        entity_id: player_clone.conn_id.as_u128() as u64,
```

Comment:
Probably should just use the u128 right? Otherwise we lose part of the uuid. Or am I missing something?

Reply:
Nvm I was confused, I see the problem now

Reply:
This is probably fine for now but we should maybe make a ticket to come up with a different solution for entity IDs since once in a blue moon these won't be unique

Reply:
Yeah I thought this would do for now, but I agree it doesn't look great